xpk/README.md

```diff
@@ -163,6 +163,41 @@ cleanup with a `Cluster Delete`.
     --cluster xpk-test
     ```
 
+* Example Workload List Output:
+
+  The below example shows four jobs of different statuses:
+
+  * `user-first-job-failed`: **filter-status** is `FINISHED` and `FAILED`.
+  * `user-second-job-success`: **filter-status** is `FINISHED` and `SUCCESSFUL`.
+  * `user-third-job-running`: **filter-status** is `RUNNING`.
+  * `user-forth-job-in-queue`: **filter-status** is `QUEUED`.
+  * `user-fifth-job-in-queue-preempted`: **filter-status** is `QUEUED`.
+
+  ```
+  Jobset Name                     Created Time           Priority   TPU VMs Needed   TPU VMs Running/Ran   TPU VMs Done   TPU Slice Dimensions   Status     Status Message                                                  Status Time
+  user-first-job-failed           2023-1-1T1:00:00Z      medium     4                4                     <none>         2xv4-8                 Finished   JobSet failed                                                   2023-1-1T1:05:00Z
```

Comment:
slice dimensions needs to be adjusted.

Reply:
removed the slice dimension since the data we were pulling is only the kueue total slice size, we dont have a tpu slice dimension without manually converting the vm count.